aya/src/util.rs

```diff
@@ -88,29 +88,48 @@ impl KernelVersion {
         (u32::from(major) << 16) + (u32::from(minor) << 8) + u32::from(patch)
     }
 
-    // This is ported from https://github.com/torvalds/linux/blob/3f01e9f/tools/lib/bpf/libbpf_probes.c#L21-L101.
-
+    // This (L92-L133) is ported from https://github.com/torvalds/linux/blob/3f01e9f/tools/lib/bpf/libbpf_probes.c#L21-L101.
```

Comment:
The line numbers in this source code will easily become outdated, I'd prefer if maybe the functions were named instead

Reply:
It's a permalink.